webapp/src/css/enketo/medic.less

```diff
@@ -8,7 +8,7 @@
  * a modal dialog.
  */
 .enketo {
-  @import "../../../dist/enketo.less";
+  @import "../../../../build/enketo.less";
```

Comment:
This change looks correct as far as keeping things in line with how they were before, but it's surprising that this `less` file should be including the other from the generated directory rather than from source.

Reply:
It's because enketo ships sass not less, so we have a build step to convert it to less and then import that. I'm sure there's a better way but this will do for now.